docs/administration-guide/databases/bigquery.md

```diff
@@ -30,3 +89,17 @@ As of version 0.30.0, Metabase tells BigQuery to interpret SQL queries as [Stand
 SELECT *
 FROM [my_dataset.my_table]
 ```
+
+## Troubleshooting
```

Comment:
We could also recommend they check out this troubleshooting guide for connecting to databases and data warehouses: https://www.metabase.com/docs/latest/troubleshooting-guide/datawarehouse.html